management_interface/management_interface/configuration.py

```diff
@@ -0,0 +1,18 @@
+import os
+from dataclasses import dataclass
+from typing import Union
+
+
+@dataclass(frozen=True)
+class Settings:
+    POSTGRES_DB: str = os.environ.get("HANS_MI_PSQL_DB", "postgres")
+    POSTGRES_USER: str = os.environ.get("HANS_MI_PSQL_USER", "postgres")
+    POSTGRES_PASSWORD: str = os.environ.get("HANS_MI_PSQL_PASSWORD", "postgres")
+    POSTGRES_HOST: str = os.environ.get("HANS_MI_PSQL_HOST", "db")
+    POSTGRES_PORT: Union[str, int] = int(os.environ.get("HANS_MI_PSQL_PORT", 5432))
+    SECRET_KEY: str = os.environ.get("HANS_MI_SECRET_KEY", "django-insecure-ys038snwqt5(l_7m%p6hh8ke20+w!8fbi+@covbk)^xl1@8r%-")
+    APP_NAME: str = os.environ.get("APP_NAME", "management_interface")
+    DEBUG: Union[str, bool] = os.environ.get("HANS_MI_DEBUG", True)
```

Comment:
Done.

Reply:
The main reason for setting this to True was to allow static files being served properly on local (or docker). I added `collectstatic` command to chain of commands, so now it may be left False by default.

Reply:
Turns out that `collectstatic` is not working as expected. I don't want to spend more time on this, I set the DEBUG in .env temporary to True to allow static files being properly served.

Reply:
I've got a quick PR for this.  Resolving this for now.